plugins/services/src/js/columns/QuotaOverviewLimitColumn.tsx

```diff
@@ -108,17 +108,17 @@ export function getLimitInfoForService(item: Service | ServiceTree): LimitInfo {
 
   return {
     limitText: getQuotaLimit({
-      count: stats.servicesCount,
+      count: stats.count,
```

Comment:
Why are we renaming this? I think that servicesCount is a lot more descriptive.

Reply:
`servicesCount` isn't accurate since it could be a Pod, Service or Task that we are counting with this variable. I wondering if there is still something we can prefix `count` with to be more descriptive, or being a part of the `ServiceQuotaRolesStats` is enough?